src/modules/DASDController.rb

```diff
@@ -465,84 +466,55 @@ def ReportActivationError(channel, ret)
+        # error report, %s is device identification
+        Report.Error(format(_("%s: Invalid status for <online>."), channel))
       when 3
-        Report.Error(
-          Builtins.sformat(
-            # error report, %1 is device identification
-            _("%1: No device found for <ccwid>."),
-            channel
-          )
-        )
+        # error report, %s is device identification
+        Report.Error(format(_("%s: No device found for <ccwid>."), channel))
       when 4
-        Report.Error(
-          Builtins.sformat(
-            # error report, %1 is device identification
-            _("%1: Could not change state of the device."),
-            channel
-          )
-        )
+        # error report, %s is device identification
+        Report.Error(format(_("%s: Could not change state of the device."), channel))
       when 5
         # https://bugzilla.novell.com/show_bug.cgi?id=446998#c15
-        Report.Error(
-          Builtins.sformat(
-            # error report, %1 is device identification
-            _("%1: Device is not a DASD."),
-            channel
-          )
-        )
+        # error report, %s is device identification
+        Report.Error(format(_("%s: Device is not a DASD."), channel))
       when 6
         # https://bugzilla.novell.com/show_bug.cgi?id=446998#c15
-        Report.Error(
-          Builtins.sformat(
-            # error report, %1 is device identification
-            _("%1: Could not load module."),
-            channel
-          )
-        )
+        # error report, %s is device identification
+        Report.Error(format(_("%s: Could not load module."), channel))
       when 7
         # http://bugzilla.novell.com/show_bug.cgi?id=561876#c8
-        Report.Error(
-          Builtins.sformat(
-            # error report, %1 is device identification
-            _("%1: Failed to activate DASD."),
-            channel
-          )
-        )
+        # error report, %s is device identification
+        Report.Error(format(_("%1: Failed to activate DASD."), channel))
       when 8
         # http://bugzilla.novell.com/show_bug.cgi?id=561876#c8
-        Report.Error(
-          Builtins.sformat(
-            # error report, %1 is device identification
-            _("%1: DASD is not formatted."),
-            channel
-          )
-        )
+        # error report, %1 is device identification
+        Report.Error(format(_("%s: DASD is not formatted."), channel))
+      when 16
+        # https://bugzilla.suse.com/show_bug.cgi?id=1091797#c8
+        headline = _("Error: channel in use")
+        # TRANSLATORS: error report, %s is device identification
+        message = format(_("%s DASD is in use and cannot be deactivated."), channel)
+        details = output_details(ret)
+
+        if details.empty?
+          Report.Error(message)
+        else
+          message << "\n\n#{_("See details for more info.")}"
+          Yast2::Popup.show(message, headline: headline, details: details)
```

Comment:
I'll to use the `Mode.auto` to decide which kind of message show, since it was requested to show the `Details...` option to give more feedback to the user.

Reply:
this is not about Mode.auto, but about blocking popup versus configurable version in Report.

Reply:
imo or schubi can give you more details about that functionality in Report module.